DiscordChatExporter.Core.Rendering/Resources/HtmlCore.css

```diff
@@ -57,6 +57,70 @@ img {
     border-radius: 3px;
 }
 
+.spoiler.hidden {
+    cursor: pointer;
+}
+
+.spoiler.hidden .spoilerText {
+    opacity: 0;
+}
+
+.spoiler-image {
+    margin-top: 0.3em;
+    border-radius: 3px;
+    -webkit-box-shadow: 0.5px 0.5px 1px 1px rgba(0,0,0,.1);
+    box-shadow: 0.5px 0.5px 1px 1px rgba(0,0,0,.1);
+    cursor: pointer;
+    overflow: hidden;
+    position: relative;
+    width: fit-content;
+}
+
+.spoiler-image:hover .spoiler-warning {
+    color: #fff;
+    background-color: rgba(0,0,0,.9);
+}
+
+.spoiler-warning {
+    color: #dcddde;
+    background-color: rgba(0,0,0,.6);
+
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    -webkit-transform: translate(-50%,-50%);
+    transform: translate(-50%,-50%);
+    text-transform: uppercase;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: column;
```

Comment:
Can drop this one too (other parts of this CSS use `flex` without vendor extensions).